book.rb

```diff
@@ -11,4 +11,16 @@ def initialize(title:, author:)
   def add_rental(rental)
     @rentals << rental
   end
+
+  def to_json(*args)
+    {
+      JSON.create_id => self.class.name,
+      'title' => @title,
+      'author' => @author
+    }.to_json(*args)
+  end
+
+  # def self.json_create(object)
+  #   new(object['title'], object['author'])
+  # end
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner

Reply:
Hi @Dipeshtwis 
Thank you for the review.
We will work on it as soon as possible.
Cheers 👍